### DISCUSSION TITLE
I Cant Login 404 page

### DISCUSSION THREAD

**Discussion by ImMrCloud** [community/community#66783] [Other Feature Feedback, Questions, & Ideas]
I cant access the 2FA website.![Image_1694448424693.jpg](https://github.com/community/community/assets/45327120/564e6f4d-ba8d-420f-862c-af6d722628ec)

I click on the sign button and redirect me to https://github.com/sessions/two factor/app and the the 404 page

**Comment by shreya-oeo** [comment]
clear the cache and open it again.

**Reply by ImMrCloud** [reply]
solved, thank you

**Reply by shreya-oeo** [reply]
Mark this as an answer:p
